lib/templates/structure/sidebar-secondary.php

```diff
@@ -5,16 +5,19 @@
  * @package Beans\Framework\Templates\Structure
  *
  * @since   1.0.0
+ * @since   1.5.0 Added ID and tabindex for skip links.
  */
 
 beans_open_markup_e(
 	'beans_sidebar_secondary',
 	'aside',
 	array(
 		'class'     => 'tm-tertiary ' . beans_get_layout_class( 'sidebar_secondary' ), // Automatically escaped.
+		'id'        => 'beans-secondary-sidebar',
```

Comment:
file is called **sidebar**-_secondary_
element is called beans_**sidebar**__secondary_
class is called **sidebar**__secondary_

but
id is called beans-_secondary_-**sidebar**

id name should follow the naming convention, right?